### PR TITLE
FORTECH-64 Add silicon styles for Slider

### PR DESCRIPTION
**Pull request by daniel-stoian-lgp**
Feature Added
Added silicon styles for Slider

Enact-DCO-1.0-Signed-off-by: Daniel Stoian daniel.stoian@lgepartner.com

### CODE REVIEW

**Comment by ybsung** [comment]
Hello @daniel-stoian-lgp 

Did you see the following screenshot? The enlarged knob is clipped in the sampler.

image

File: styles/variables-carbon.less [hunk 1]
```diff
@@ -78,7 +78,7 @@
 
 // Slider
 // ---------------------------------------
-@agate-slider-knob-active-transform: @agate-translate-center scale(1);
+@agate-slider-active-knob-transform: @agate-translate-center scale(1);
```
Comment (ybsung) [inline styles/variables-carbon.less:81]: Could we omit `scale(1)`? In my opinion, by default, the scale would be `1`. Please fix me if I'm wrong.

Comment (ybsung) [reply styles/variables-carbon.less:81]: If we have to use the `scale`, please use scale`(@agate-slider-knob-resting-state-scale)`

Comment (daniel-stoian-lgp) [reply styles/variables-carbon.less:81]: I removed the scale. it was not necessary

**Pull request merged**